build.sbt

```diff
@@ -183,24 +181,6 @@ lazy val catsEffect = (project in file("cats"))
   )
   .dependsOn(formats, scanamo, testkit % "test->test")
 
-lazy val scalaz = (project in file("scalaz"))
-  .settings(
-    name := "scanamo-scalaz",
-    commonSettings,
-    publishingSettings,
-    libraryDependencies ++= List(
-      awsDynamoDB,
-      "com.codecommit" %% "shims"           % shimsVersion,
```

Comment:
I think `shimsVersion` was only used here.

Reply:
Correct and removed :)